ocaml/tools/build_jane_syntax_with_active_opam_switch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Build Jane Syntax files with the opam switch's OCaml. This verifies
+# that they can be put into public release with minimal changes;
+# see Note [Buildable with upstream] in ocaml/parsing/jane_syntax.mli
+
+cd $(git rev-parse --show-toplevel)/ocaml
+
+files_in_dependency_order=(
+  utils/language_extension_kernel.{mli,ml}
+  language_extension.ml
+  parsing/jane_asttypes.mli
+  parsing/{jane_syntax_parsing,jane_syntax}.{mli,ml}
+)
+
+function basenames_in_dependency_order() {
+  echo "${files_in_dependency_order[@]}" | xargs -n 1 basename
+}
+
+# Copy files to another directory before compiling so we're confident
+# we're not using the build artifacts of an earlier build.
+tmp=$(mktemp -d)
+cp "${files_in_dependency_order[@]}" "$tmp" 2>/dev/null || true
```

Comment:
Why is this silencing errors?  As in both:
1. What errors could happen?
2. Why wouldn't we want to see them?

Reply:
This was a hack to get around the fact that `language_extensions.ml` doesn't exist in the source directory. (That's both (1) an error and (2) an error I don't care to see.) It's a bit more explicit to just filter it out so I did that instead.